src/App.js

```diff
@@ -1,25 +1,69 @@
 import React from 'react';
+import { Clock } from './components/Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 'null',
```

Comment:
Здесь немного не согласен. Почему решил для имени задать стартовое значение null с типом данных строка. Лучше задай начальное значение 0.

Reply:
Vadym is right. Don't change types. Better use some initial value

```suggestion
    clockName: `${Math.ceil(Math.random() * 100)}`,
```